postgresql/config/functions.sh

```diff
@@ -3,13 +3,6 @@ init_pgpass() {
   chmod 0600 {{pkg.svc_config_path}}/.pgpass
```

Comment:
Is `write_wale_env()` still required in this file?

Reply:
Nope. I can clean that out!